verify.ts

```diff
@@ -0,0 +1,265 @@
+            break;
+        }
+
+        spinner = ora();
+        spinner.start(`${contractName} at ${addresses[contractName]}`);
+
+        let foundIn = './node_modules/@daostack/arc';
+        process.chdir(path.join(__dirname, foundIn));
+
+        let solFilePath = glob.sync(`./contracts/**/${contractName}.sol`);
+        if (!solFilePath.length) {
+          foundIn = './node_modules/@daostack/arc/node_modules/@daostack/infra';
+          process.chdir(path.join(__dirname, foundIn));
+          solFilePath = glob.sync(`./contracts/**/${contractName}.sol`);
+        }
+
+        if (!solFilePath.length) {
+          foundIn = './node_modules/@daostack/arc-hive';
+          process.chdir(path.join(__dirname, foundIn));
+          solFilePath = glob.sync(`./contracts/**/${contractName}.sol`);
+        }
+
+        if (!solFilePath.length) {
+          spinner.fail(`contract ${contractName}.sol not found`);
+          continue;
+        }
+
+        const flattened = await truffleFlattener([solFilePath[0]]);
+
+        if (options.outputFlattened) {
+          fs.writeFileSync(path.join(options.outputFlattened, `flattened.${contractName}.sol`), flattened);
+        }
+
+        // tslint:disable-next-line: max-line-length
+        let version = require(`${foundIn}/build/contracts/${contractName}.json`).compiler.version;
+        version = `v${version.substr(0, version.indexOf('Emscripten') - 1)}`;
+
+        const apiConfig = {
+          action: 'verifysourcecode',
+          apikey: APIKEY,
+          compilerVersion: version,
+          constructorArguements: constructorArguments ? constructorArguments.toString('hex') : '',
+          contractaddress: addresses[contractName],
+          contractname: contractName,
+          module: 'contract',
+          optimizationUsed: '1',
+          runs: '200',
+          sourceCode: flattened,
+        };
+
+        // console.dir(Object.assign({}, apiConfig, { sourceCode: `${flattened.substr(0, 50)}...` }));
+
+        const encodedPostData = querystring.stringify(apiConfig).replace(/%20/g, '+');
+        const response = await axios.post(etherscanUrl, encodedPostData,
+          {
+            headers: {
+              'Content-Length': encodedPostData.length,
+              'Content-Type': 'application/x-www-form-urlencoded',
+            },
+          }
+        );
+
+        const result = response.data;
+
+        if (result.status === '1') {
+          // 1 = submission success, use the guid returned (result.result) to check the status of the submission.
+          anySucceeded = true;
+          const verifyResult = await validateResult(result.result, etherscanUrl);
+          if (verifyResult.result !== 'Pending in queue') {
+            // tslint:disable-next-line: max-line-length
+            spinner.fail(`${contractName} at ${addresses[contractName]} ${verifyResult.result}, GUID: ${result.result}`);
+          } else {
+            // tslint:disable-next-line: max-line-length
```

Comment:
why to disable that here?

Reply:
i removed it and split the following line.